python/lib/db/base.py

```diff
@@ -0,0 +1,5 @@
+from sqlalchemy.orm import DeclarativeBase
+
+
+class Base(DeclarativeBase):
+    pass
```

Comment:
could you enlighten me on the purpose of this class? I am sure it has a purpose but right now, I am not sure which one 😉

Reply:
Basically in SQLAlchemy, all the ORM classes of a given schema must inherit from a single "base" class (this one), which must itself inherit from the `DeclarativeBase` SQLAlchemy class. That's just how the library registers the ORM classes.

Reply:
sweet! Thank you for the explanation :)